bluepyemodel/access_point/local.py

```diff
@@ -154,14 +192,48 @@ def set_emodel(self, emodel):
 
     def load_pipeline_settings(self):
         """ """
-
-        settings = self.get_recipes().get("pipeline_settings", {})
-
+        recipes = self.get_recipes()
+        settings = recipes.get("pipeline_settings", {})
+        if isinstance(settings, str):
+            # read the pipeline settings from file
+            settings = self.get_json("pipeline_settings")
         if "morph_modifiers" not in settings:
-            settings["morph_modifiers"] = self.get_recipes().get("morph_modifiers", None)
-
+            settings["morph_modifiers"] = recipes.get("morph_modifiers", None)
         return EModelPipelineSettings(**settings)
 
+    def _config_to_final(self, config):
+        """Convert the configuration stored in EM_*.json to the format used for final.json."""
+        metadata = self.emodel_metadata
+        return {
+            metadata.emodel: {
```

Comment:
Sounds good. Alternatively, would it make sense to add these keys to the `EM_*.json` file, so that all the keys can be retrieved from this file?
It would be a step forward to being able to initialize LocalAccessPoint with just `recipes.json` (without manually specifying etype, emodel, iteration)

Reply:
Not really...
BluePyEModel has been designed to separate the resource data in the `EM_*.json` from the metadata (etype, emodel, iteration, etc.). We expect the user to always give the metadata when creating the access point, which will then fetch the appropriate files form the file system for the local access point or from nexus for the nexus access point.

Reply:
In the use case I'm considering, I'm retrieving all the information of the EModel from Nexus and storing them locally.
Then, in a separate step, I want to load the EModel from the local data without querying Nexus anymore.
If the metadata aren't stored in any local file, it won't be possible to load the EModel with just that information, so it seems that the information retrieved from Nexus and stored locally would be incomplete without the metadata.

Alternatively, I can save the metadata from Nexus to an additional file, and I could add for example a function or a class method `LocalAccessPoint.from_metadata()` to load this file and initialize the LocalAccessPoint.
This file would act as the entry point for my use case, as it is circuit_config.json for circuits, or simulation_config.json for simulations, or config.json for simulation campaigns.

Reply:
Saving the metadata from nexus in an additional file and using it as an entry point sounds fine to me